store/gcloud/gcloud_store.go

```diff
@@ -0,0 +1,44 @@
+package gcloud
+
+import (
+	"context"
+	"fmt"
+	"github.com/libopenstorage/secrets/gcloud"
+
+	"github.com/libopenstorage/secrets"
+	"github.com/libopenstorage/secrets/store"
+)
+
+type gcloudStore struct {
+	secrets secrets.Secrets
+}
+
+func (s *gcloudStore) Get(ctx context.Context, key store.SecretKey) (secret map[string]any, err error) {
+	secretId := createSecretId(key)
+	keyContext := map[string]string{secrets.CustomSecretData: "true"}
```

Comment:
Lets remove the gcloud implementation as this keyContext is interpreted in many other ways in gcloud.